administration/src/cards/PdfFactory.ts

```diff
@@ -95,49 +92,27 @@ Aussteller: ${region.prefix} ${region.name}`,
       size: 10,
     }
   )
-
-  if (staticVerificationCode) {
-    doc.addPage()
-    const qrCodeText2 = uint8ArrayToBase64(
-      new QrCode({
-        qrCode: {
-          value: staticVerificationCode,
-          case: 'staticVerificationCode',
-        },
-      }).toBinary()
-    )
-    drawjsPDF(qrCodeText2, qrCodeX, qrCodeY, qrCodeSize, doc)
-  }
 }
 
-function fillCodeArea(
-  activationCode:
-    | {
-        value: DynamicActivationCode
-        case: 'dynamicActivationCode'
-      }
-    | {
-        value: StaticVerifyCode
-        case: 'staticVerifyCode'
-      },
-  x: number,
-  y: number,
-  size: number,
-  page: PDFPage
-) {
+function fillCodeArea(qrCode: PdfQrCode, x: number, y: number, size: number, page: PDFPage) {
   const qrCodeSizePdf = mmToPt(size)
   const qrCodeXPdf = mmToPt(x)
   const qrCodeYPdf = page.getSize().height - qrCodeSizePdf - mmToPt(y)
 
-  const dynamicQrCode = uint8ArrayToBase64(
+  const qrCodeContent = uint8ArrayToBase64(
     new QrCode({
-      qrCode: activationCode,
+      qrCode: qrCode,
     }).toBinary()
   )
-  drawQRCode(dynamicQrCode, qrCodeXPdf, qrCodeYPdf, qrCodeSizePdf, page)
+  drawQRCode(qrCodeContent, qrCodeXPdf, qrCodeYPdf, qrCodeSizePdf, page, false)
 }
 
-export async function generatePdf(activationCodes: DynamicActivationCode[], region: Region, pdfConfig: PdfConfig) {
+export async function generatePdf(
+  activationCodes: DynamicActivationCode[],
+  staticCodes: StaticVerificationCode[] | null,
```

Comment:
Can we change the signature to something like `cards: { dynamicActivationCode: DynamicActivationCode, staticVerificationCode | null }[]` ?

If not, please add a check that the arrays have the same length